persistence-cassandra-3.11/src/main/java/io/stargate/db/cassandra/CassandraPersistenceActivator.java

```diff
@@ -36,7 +36,7 @@
 import org.apache.cassandra.dht.Murmur3Partitioner;
 import org.apache.cassandra.io.util.FileUtils;
 import org.apache.cassandra.locator.SimpleSnitch;
-import org.apache.cassandra.metrics.CassandraMetricsRegistry;
+import org.apache.cassandra.metrics.Cassandra311MetricsRegistry;
```

Comment:
Had to rename this because jacoco doesn't like multiple classes with the same name.

Reply:
There are probably other instances of this problem across the 3 persistence implementations.

Reply:
Yes, extensions tends to reuse names. And frankly, I'd prefer we avoid the route of renaming all the classes just because a tool is deficient.